doc/enterprise/flight_recorder.rst

```diff
@@ -1,218 +1,74 @@
-.. _enterprise-flight-recorder:
+..  _enterprise-flight-recorder:
 
 Flight recorder
 ===============
 
+**Example on GitHub**: `flightrec <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/config/instances.enabled/flightrec>`_
+
 The flight recorder available in the Enterprise Edition is an event collection tool that
 gathers various information about a working Tarantool instance, such as:
 
 *   logs
-
 *   metrics
-
 *   requests and responses
 
 This information helps you investigate incidents related
 to :ref:`crashing <admin-disaster_recovery>` a Tarantool instance.
 
-
-.. _enable:
+.. _enterprise-flight-recorder_enable:
 
 Enabling the flight recorder
 ----------------------------
 
 The flight recorder is disabled by default and can be enabled and configured for
 a specific Tarantool instance.
-To enable the flight recorder, set the ``flightrec_enabled``
-:doc:`configuration option </reference/configuration/index>` to ``true``.
-This option is :ref:`dynamic <box_cfg_params>` and can be changed at runtime by calling ``box.cfg{}``:
+To enable the flight recorder, set the :ref:`flightrec.enabled <configuration_reference_flightrec_enabled>`
+configuration option to ``true``.
 
-.. code-block:: lua
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/flightrec/config.yaml
+    :language: yaml
+    :start-at: flightrec:
+    :end-at: enabled: true
+    :dedent:
 
-    box.cfg{ flightrec_enabled = true }
 
 After ``flightrec_enabled`` is set to ``true``, the flight recorder starts collecting data in the flight recording file  ``current.ttfr``.
-This file is stored in the :ref:`memtx_dir <cfg_basic-memtx_dir>` directory.
+This file is stored in the ``memtx.dir`` directory.
```

Comment:
Please double check that this is true. As I see, `snapshot.dir` is used in a new confifuration:

https://www.tarantool.io/en/doc/latest/concepts/configuration/#snapshots-and-write-ahead-logs